src/errors.jl

```diff
@@ -86,6 +86,14 @@ function gen_analysis(io, ex::DistributionNotImplemented)
     end
 end
 
+function try_show_method_candidates(io, args...)
+    try
+        Base.show_method_candidates(io, args...) # this isn't exported, so it might break
+    catch ex
+        @debug("Unable to show method candidates:\n$(sprint(showerror, ex))")
```

Comment:
Maybe this should say "Please submit an issue"?

Reply:
done